src/pages/courses.tsx

```diff
@@ -14,6 +14,7 @@ import { FaPlus } from "react-icons/fa";
 
```

Comment:
unused imports in this file

Reply:
Oh sorry! I guess I should be using some checker with my VScode

Reply:
Seems like this one is used in add course button

Reply:
Oh sorry I got it. There were other unused imports. Fixed them!

Reply:
Yeah GitHub wouldn't let me include the actual lines in this comment, the import here is definitely used